src/assets/styles/css/layout.css

```diff
@@ -0,0 +1,431 @@
+}
+
+body::-webkit-scrollbar {
+  width: 1em;
+}
+
+body::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+
+body::-webkit-scrollbar-thumb {
+  background-color: darkgrey;
+  outline: 1px solid slategrey;
+}
+
+.portfolio-container {
+  height: 200vh;
+  width: 100vw;
+  background: url("../../img/wall-paper.jpg") center center fixed;
+  background-size: cover;
+}
+
+.welcome-screen {
+  background-color: rgba(0, 0, 0, 0.6);
+  height: 100vh;
+  width: 100%;
+  color: rgb(255, 255, 255);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
+  line-height: 50px;
+}
+
+.welcome-screen button {
+  width: 6rem;
+  margin: 10rem auto;
+  height: 6rem;
+  background-color: rgb(81, 11, 95);
+  color: rgb(255, 255, 255);
+  font-size: 1.2em;
+  border: none;
+  border-radius: 150px;
+  outline: none;
+  cursor: pointer;
+  transition: linear 0.5s;
+  box-shadow: 15px 15px 15px rgba(0, 0, 0, 0.3);
+}
+
+.welcome-screen button:hover {
+  transform: scale(1.1);
+  background-color: rgba(81, 11, 95, 0.5);
+}
+
+.portfolio-navigation {
+  width: 95%;
+  height: 90vh;
+  margin: 2rem auto;
+  background-color: rgba(0, 0, 0, 0.8);
+  border-radius: 0.5rem;
+  box-shadow: 1rem 1rem 1rem rgba(0, 0, 0, 0.2);
+}
+
+.top-nav {
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  height: 5%;
+  width: 100%;
+  color: rgb(0, 0, 0);
+  background-color: rgb(215, 222, 246);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.portfolio-navigation ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.portfolio-navigation li {
+  margin-right: 1rem;
+}
+
+.close-btn {
+  height: 20px;
+  width: 20px;
+  border-radius: 100%;
+  background-color: rgb(237, 93, 87);
+}
+
+.minimize-btn {
+  height: 20px;
+  width: 20px;
+  border-radius: 100%;
+  background-color: rgb(249, 192, 61);
+}
+
+.maxmize-btn {
+  height: 20px;
+  width: 20px;
+  border-radius: 100%;
+  background-color: rgb(100, 206, 66);
+}
+
+main {
+  display: grid;
+  grid-template: 1fr / 1fr 5fr;
+  height: 95%;
+}
+
+.aside,
+section {
+  height: 100%;
+  width: 100%;
+}
+
+.aside {
+  background-color: rgba(59, 59, 59, 0.1);
+  border-bottom-left-radius: 0.5rem;
+}
+
+section {
+  background-color: rgb(51, 51, 51);
+  border-bottom-right-radius: 0.5rem;
+}
+
+figure > div {
+  display: flex;
+  flex-direction: column;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  text-align: justify;
+  justify-content: center;
+  height: 100%;
+}
+
+.aside button {
+  width: 100%;
+  border: none;
+  margin-top: 0.2rem;
+  padding: 1rem;
+  cursor: pointer;
+  border-top-left-radius: 1rem;
+  border-bottom-left-radius: 1rem;
+  margin-left: 1rem;
+  outline: none;
+}
+
+.aside button:hover {
+  background-color: rgba(72, 72, 72, 0.5);
+  color: rgb(255, 255, 255);
+}
+
+.dark-bg {
+  background-color: rgb(24,26,27);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)